website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -113,10 +113,196 @@ func init() {
 * After adding new import paths to your operator project, run `go mod vendor` if a `vendor/` directory is present in the root of your project directory to fulfill these dependencies.
 * Your 3rd party resource needs to be added before add the controller in `"Setup all Controllers"`.
 
-### Metrics
+### Monitoring and Observability
+This section aims to explain how to expose the first operator's custom metrics, [alerts] and [recording rules]. It focuses on the technical aspects, and demonstrates the implementation over the sample [memcached-operator]. 
```

Comment:
Nit: What is the "first operator" in this context? IMO something like:
```suggestion
This section covers how to create custom metrics, [alerts] and [recording rules] for your operator. It focuses on the technical aspects, and demonstrates the implementation by updating the [memcached-operator] created in the [tutorial]. 
```
sounds a bit better

Reply:
Done, thanks